### DISCUSSION TITLE
None of the .fla files for FNF will open and Adobe Animate keeps saying an error has occured.

### DISCUSSION THREAD

**Discussion by lissnomsnail** [FunkinCrew/Funkin#3905] [Q&A]
I know nothing about this stuff and I haven't modded FNF in forever.

I have Adobe Animate 2023 and I can't update it because I pirated it (I'm poor, fuck off) 

It worked previously when I had Adobe Animate 2021 so I think me getting a newer version is the reason why it doesn't work now.

How can I fix this since I want to get back into modding FNF?

![Screenshot (136)](https://github.com/user-attachments/assets/452c52bf-24f3-4748-9520-a2f71eafa0b2)

**Comment by moxie-coder** [comment]
to be fair pretty sure anyone would want to pirate adobe at this point
